package.json

```diff
@@ -121,7 +121,7 @@
     "libp2p-circuit": "~0.1.4",
     "libp2p-floodsub": "~0.13.1",
     "libp2p-kad-dht": "~0.6.0",
-    "libp2p-mdns": "~0.9.1",
+    "libp2p-mdns": "^0.9.2",
```

Comment:
keep the use of `~`

Reply:
oops, that was `npm install` and poor review by me.  Will fix.